src/WorldState.Data/Models/Alert.cs

```diff
@@ -19,12 +19,6 @@ public class Alert : ITimeSensitive
         [JsonProperty]
         public AlertMission Mission { get; private set; }
 
-        [JsonProperty("active")]
-        public bool IsActive { get; private set; }
```

Comment:
you probably don't want to remove these if you're trying to keep parity with the api, as these will be present for alerts that are present in the worldstate before or after they're active.

Reply:
Do you mean that, in some circumstances, the activation and expiration timestamp may be incorrect or unknown?

I have added extensions (in https://github.com/WFCD/Warframe.NET/pull/26/commits/960a8d249ae3ee71f6599fed5d40fbc89b6749bf) to give everything that has both activation and expiration timestamps `IsActive()`, `HasExpired()` and `GetTimeRemaining()`. as long as both timestamps are correct, these methods return the current event status according to user's local (machine) date/time settings.

Reply:
uh, no, cause that's not what i said.... I'm saying that if an alert or event is not actually active yet, that's not always immediately apparent unless you're going to keep something indicating if it is... but looks like you added some methods for that.